### PR TITLE
[Port-ish]Buildable shutters, filling cabinet recipe and dedicated category for crafting structures

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

**CURRENTLY ON HOLD AS THERE HASN'T BEEN ANY MAP UPDATE, AS I AM WAITING FOR THE GREP PILE PORT TO BE MERGED**

This PR ports https://github.com/tgstation/tgstation/pull/56486, https://github.com/tgstation/tgstation/pull/61746, https://github.com/tgstation/tgstation/pull/58231 and implement a dedicated crafting tab for structures, where now stuff like the guillotine, curtains and the buildable shutters.

Future to do:
- [x] Port or sprite anew the Radiation and the window shutter off https://github.com/tgstation/tgstation/pull/53861 and add their respective recipe 

## Why It's Good For The Game

More customization options, Ability to make your own "safe" area, fun for everyone!

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

dreamseeker_GogHoT8gGP

dreamseeker_oDHQPwP8sh

https://user-images.githubusercontent.com/75247747/193803033-c4ddd06a-7dbb-4e1b-a0ef-75be93bca785.mp4

</details>

## Changelog
:cl:
add: Added craftable filing cabinets that can be constructed for 2 iron sheets
add: Added Rad-resistant shutter and Window shutter
add: Added Radproof shutters next to the SM on all the maps.
add: Added blast doors and shutters with their variants to the crafting menu and blast door 
add: Replaced all the old Supermatter shutters with radiation proof one on all the maps
add: Brand new sounds from Tau Ceti for shutters, no more the same sounds as heavy duty blast doors
del: Removed old unused multitile shutters
controllers to autolathes. As well as deconstruction steps
tweak: new dedicated structures category, instead of misc for all structure things
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/shutters.dm [hunk 1]
```diff
@@ -6,12 +6,68 @@
...
+
+/obj/machinery/door/poddoor/shutters/update_icon()
+	if(density)
+		icon_state = "[base_state]_closed"
+	else
+		icon_state = "[base_state]_open"
+
+/obj/machinery/door/poddoor/shutters/radiation
+	name = "radiation shutters"
+	desc = "Depleted uranium-lined shutters with a radiation hazard symbol. Whilst this won't stop you getting irradiated, especially by a supermatter crystal, it will stop the majority of radiation travelling as far."
```
Comment (itsmeow) [inline code/game/machinery/doors/shutters.dm]: Remove "travelling as far"

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -746,6 +747,7 @@ GLOBAL_LIST_INIT(bronze_recipes, list ( \
 	. = ..()
 	pixel_x = 0
 	pixel_y = 0
+/datum/crafting_recipe/aquarium
```
Comment (itsmeow) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: looks like a mistake

File: code/game/objects/structures/watercloset.dm [hunk 3]
```diff
@@ -388,7 +388,7 @@
 
 
 /obj/structure/curtain
-	name = "curtain"
+	name = "Plastic curtain"
```
Comment (itsmeow) [inline code/game/objects/structures/watercloset.dm]: lowercase

File: code/modules/assembly/doorcontrol.dm [hunk 4]
```diff
@@ -13,6 +13,12 @@
 	if(id)
 		. += "<span class='notice'>Its channel ID is '[id]'.</span>"
 
+/obj/item/assembly/control/multitool_act(mob/living/user)
+	var/change_id = input("Set the shutters/blast door/blast door controllers ID. It must be a number between 1 and 100.", "ID", id) as num|null
```
Comment (itsmeow) [inline code/modules/assembly/doorcontrol.dm]: `controller's`. Also says blast door twice.

File: code/game/machinery/doors/poddoor.dm [hunk 5]
```diff
@@ -14,7 +14,62 @@
...
+		if(density)
+			to_chat(user, "<span class='warning'>You need to open [src] before opening its maintenance panel.</span>")
+			return
+		else if(default_deconstruction_screwdriver(user, icon_state, icon_state, W))
+			to_chat(user, "<span class='notice'>You [panel_open ? "open" : "close"] the maintenance hatch of [src].</span>")
+			return TRUE
+
+	if(panel_open)
+		if(W.tool_behaviour == TOOL_MULTITOOL && deconstruction == BLASTDOOR_FINISHED)
+			var/change_id = input("Set the shutters/blast door/blast door controllers ID. It must be a number between 1 and 100.", "ID", id) as num|null
```
Comment (itsmeow) [inline code/game/machinery/doors/poddoor.dm]: see other review

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by itsmeow** [comment]
Are you able to put the controllers into button casings?

**Comment by PowerfulBacon** [comment]
Are shutters even breakable conventionally? Can you defeat them?

**Comment by PestoVerde322** [comment]
> Are you able to put the controllers into button casings?

Absolutely, you can build them by an autolathe, however I am considering to toss a bunch of them in tool storage and in engineering, or maybe in the vending machine for parts.

**Comment by PestoVerde322** [comment]
> Are shutters even breakable conventionally? Can you defeat them?

Depends, unless specified with the indestructible variant, they they're breakable by default (and constructible by "crafting" them as if you're building a guillotine).

**Comment by itsmeow** [comment]
Also would appreciate if this was put on the Guide to Construction wiki page

**Comment by PestoVerde322** [comment]
> Also would appreciate if this was put on the Guide to Construction wiki page

Updated the wiki with the recipes for the shutters.

**Comment by PestoVerde322** [comment]
Also note to self, remove Flandstation from this, since Sp5 is in it's way

**Comment by PestoVerde322** [comment]
and rollback'd

**Review by itsmeow — CHANGES_REQUESTED** [review]
move aquarium to construction category

**Comment by itsmeow** [comment]
What are the mapping changes?

**Comment by PestoVerde322** [comment]
> What are the mapping changes?

ah damn i forgot to add it in the CL, but it adds on all the maps but fland radshutters next to the SM.

**Review by itsmeow — APPROVED** [review]

**Pull request merged**